### ISSUE TITLE
Moodle plugin | no permission

### ISSUE THREAD

**Issue by bobanmak** [ether/etherpad-lite#2842]
I'm using the latest Etherpad-Lite Plugin for Moodle but i'm getting this massage, that i do not have the permission to open this pad. It looks like this:
![moodle](https://cloud.githubusercontent.com/assets/12385796/11421569/0e99ab6e-9436-11e5-82bb-cac3cc221c2d.JPG)
My settings.json file looks like this:
![sett](https://cloud.githubusercontent.com/assets/12385796/11421862/7e37197e-9437-11e5-902e-e7e83631c157.png)
 Am i missing something or this moodle plugin is no more supported?

**Comment by lpagliari** [comment]
I've never worked with Moodle, so I don't know how it is configured to work with Etherpad. But usually you get this message when your settings.json has `editOnly` set to `true` (as it is in the screenshot you've shared).

Be aware that by setting `editOnly` to `false` you'll allow any user to create pads anytime they want, so this change on your `settings.json` might be risky, depending on how Moodle works with Etherpad.

I hope that helps.

**Comment by bobanmak** [comment]
Hi there. Thanks for the fast response. 
I've tried setting the `editOnly` parameter to `false`, but i'm still getting the same response.
**edit
I was able to fix this problem. The configuration of the moodle plugin was incorrect. The Cookie Domain that i used, had the complete adresse with the subdomain and not just the single domain name.
![cook](https://cloud.githubusercontent.com/assets/12385796/11426190/7d0fb20e-945a-11e5-81f3-209c04393e7f.JPG)
